src/frontend/src/pages/GanttPage/GanttChartPage.tsx

```diff
@@ -351,18 +299,22 @@ const GanttChartPage: FC = () => {
         <GanttChart
           startDate={startDate}
           endDate={endDate}
-          teamsList={sortedTeamList}
-          teamNameToGanttTasksMap={teamNameToGanttTasksMap}
+          teamsList={teamList.sort((a, b) => sortTeamList(a, b, defaultGanttFilters, searchText))}
           saveChanges={saveChanges}
           showWorkPackagesMap={showWorkPackagesMap}
           setShowWorkPackagesMap={setShowWorkPackagesMap}
           highlightedChange={ganttTaskChanges[ganttTaskChanges.length - 1]}
           addProject={addProjectHandler}
           addWorkPackage={addWorkPackageHandler}
           getNewProjectNumber={getNewProjectNumber}
+          getNewWorkPackageNumber={getNewWorkPackageNumber}
+          defaultGanttFilters={defaultGanttFilters}
+          searchText={searchText}
```

Comment:
a bit extra that these two need to get passed down, but I see why you're doing it. A lot of logic/ state is packed into GanttPage rn, would be worth seeing if we can distribute it a bit more if possible in the future, but this looks good for now

Reply:
Yeah i tried with having the project state be int he team section. Another thing we have to do is enforce only one team being edited at a time, because pressing save will wipe out data for other edited teams. Might be worth moving all of this information into the teams sections and then move the modals down into there too